aqua/kitty/Portfile

```diff
@@ -47,7 +47,7 @@ depends_lib-append  port:harfbuzz
 
 build.cmd           "${python.bin} setup.py"
 build.target        kitty.app
-build.args-append   --verbose
+build.args-append   --verbose --update-check-interval=0
```

Comment:
Does this disable kitty checking for updates?

Reply:
Yes, it is. 

From [setup.py](https://github.com/kovidgoyal/kitty/blob/cdf64bf01614324200eac8955503e9a3ac432f73/setup.py#L1202-L1207):

> When building a package, the default value for the update_check_interval setting will be set to this number. Use zero to disable update checking.

Reply:
OK np